backend/src/repositories/BlocksAuditsRepository.ts

```diff
@@ -78,6 +78,7 @@ class BlocksAuditRepositories {
           blocks_audits.height,
           blocks_audits.hash as id,
           UNIX_TIMESTAMP(blocks_audits.time) as timestamp,
+          UNIX_TIMESTAMP(blocks_audits.first_seen) as firstSeen,
```

Comment:
`firstSeen` comes out as a string here. I think we're safe to convert to a javascript number before returning the final object, even with microsecond precision.